mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -619,10 +624,26 @@ class WidgetAdapter(
         }
 
         override fun onStartTrackingTouch(slider: Slider) {
-            // no-op
+            isTracking = true
         }
 
         override fun onStopTrackingTouch(slider: Slider) {
+            isTracking = false
+            sendUpdate()
+        }
+
+        override fun onValueChange(slider: Slider, value: Float, fromUser: Boolean) {
+            if (!fromUser) {
+                return
+            }
+            Log.e(TAG, "${System.currentTimeMillis() - lastChange}")
+            if (System.currentTimeMillis() - lastChange < 1000) {
```

Comment:
I don't really like this because it will send updates at essentially random times ... I added an alternative approach/suggestion to #3116